stubs/flake8-bugbear/bugbear.pyi

```diff
@@ -0,0 +1,26 @@
+import argparse
+import ast
+from typing import Any
+
+class BugBearChecker:
+    name: str
+    version: str
+    tree: ast.AST | None
+    filename: str | None
```

Comment:
`filename` is not intended to ever be `None`, because the default value is `"(none)"`. Same goes for `max_line_length`, and also the corresponding `__init__` arguments.